client/src/view/info-pages.tsx

```diff
@@ -54,25 +30,20 @@ export const INFO_PAGES: JSX.Element[] = [
     (
         <div>
             <p>
-                As the story goes, <strong>galapa</strong> means something like
-                "family" in the local dialect and a <strong>hexalot</strong>,
-                is a unique beehive pattern of surface spots where a *gotchi* lives.
+                Every "hexalot" is a place with a completely unique pattern of spots
+                and it is forever connected to its neighbors.
             </p>
             <p>
-                Every *hexalot* is a place with a completely unique pattern
-                and it is forever connected to its neighbors.
+                A "gotchi" emerges from a tower on the middle spot of a hexalot,
+                evolving first body shape and then muscle coordination.
             </p>
         </div>
     )
     ,
     (
         <div>
             <p>
-                A <strong>galapagotchi</strong> emerges from its hexalot,
-                evolving first body shape and then muscle coordination.
-            </p>
-            <p>
-                The genes of a galapagotchi are not DNA like ours, but are instead frozen sequences of dice.
+                The genes of a gotchi are not DNA like ours, but are instead frozen sequences of dice.
```

Comment:
Might wanna add a comment about how our DNA is encoded with ACTG, but gotchi genes are encoded with 123456.

Reply:
```suggestion
                The genes of a gotchi are not DNA like ours, but are instead sequences of dice rolls.
```

Or something

Reply:
did

Reply:
A "gotchi" is a virtual animal which is born from an egg in the tower in the middle of a hexalot, evolving first body shape and then muscle coordination.

Reply:
LOL, how to say it?!

Reply:
https://www.youtube.com/watch?v=f3t9SfrfDZM